sphinx-docs/.screenshot_dump/.gitignore

```diff
@@ -1,3 +0,0 @@
-# Screenshots are dumped here by the build process
-*
```

Comment:
Why'd ya remove this? git won't include empty directories, so this file ensures the directory exists but nothing in it is committed.